### ISSUE TITLE
Add option to color light chip cards based on state

### ISSUE THREAD

**Issue by wac-** [piitaya/lovelace-mushroom#101]
Setting a chip card for a light to "Name" for disambiguation unfortunately makes it impossible to know the state. If the icon was colored the same way it is in the light card chips can be colored that would be quickly visual way to indicate state.

**Comment by piitaya** [comment]
You can set the `color` for entity chip via the editor 😊

**Comment by wac-** [comment]
Sure, but hard-coding the color in the editor is a bit different than having the icon color follow the state of the light like how it is done at https://github.com/piitaya/lovelace-mushroom/blob/main/src/cards/light-card/light-card.ts#L158

**Comment by piitaya** [comment]
Le rules are the same for card and chip. The color is applied only when active.

**Comment by fgonza2** [comment]
what if the value is numeric, say for example i want a custom icon color when is not 0%, is that possible? meaning a 0 value gives me the gray dim color and a non-zero value of the entity gives you the icon_color

**Comment by wac-** [comment]
I've got a [work in progress over in a fork](https://github.com/wac-/lovelace-mushroom/blob/color-light-entity-chip/src/cards/chips-card/chips/entity-chip.ts) that causes the icon to follow the device color and state the same way the light card does, if chip has `icon_color: device_color` manually set in the dashboard config.

I am struggling to make `device_color` a setting in the editor interface, but there's probably some nuance to those menus that I'm just not getting right now.

I've started thinking maybe a dedicated light-chip rather than using a more generic entity-chip would be the right choice for all this. That could also have default action of "toggle" which seems more intuitive as a default tap on a light action.
